adapters/socketio_win32.c

```diff
@@ -331,9 +339,14 @@ CONCRETE_IO_HANDLE socketio_create(void* io_create_parameters)
                 else
```

Comment:
>else [](start = 16, length = 4)

Nit: the number of nested else's is getting deep.  Any chance we can move more to the 
else if (dosomething() == error)
{
   fail
}
else if (doNextSomething == error) 
{
...
}

Where it's a bit flatter?

Reply:
I like that way too. Shall be done.

Reply:
Done.